README.md

```diff
@@ -1,23 +1,24 @@
-# Accurate Instagram Filters Reconstruction
+# Accurate Image Filters Reconstruction
 
 There's a bunch of apps out there (e.g., Instagram) allowing you to apply color filters to images.
 You might be interested in cloning their behavior: reconstruct them. This
 repository holds tools for semi-automatically and very accurately reconstructing
 color filters.
 
-The truth is folks like Instagram filters. They are trying to
+The truth is folks like filters. They are trying to
 [reproduce](https://github.com/girliemac/Filterous)
 them [again](https://github.com/girliemac/filterous-2)
 and [again](https://github.com/acoomans/instagram-filters).
 And [again](https://github.com/lukexyz/CV-Instagram-Filters)
 and [again](https://www.practicepython.org/blog/2016/12/20/instagram-filters-python.html).
 And [again](https://code.tutsplus.com/tutorials/create-instagram-filters-with-php--net-24504)
 and [again](https://picturepan2.github.io/instagram.css/).
+
 The problem with the attempts is they mostly deal with manually correcting
 colors.
+While, there's just a single robust method of color reconstruction.
```

Comment:
The actual meaning was "**This is** the only accurate reconstruction", not "there is just a single"

Reply:
"This is the only" implies there is nothing around but the thing we're talking about, which equals to "Just a single <something>". Is it the only accurate reconstruction, really?

Reply:
"This is the only" is more related to attempt, than to the method. Who knows which other methods are exist.